### ISSUE TITLE
Parts of the documentation still link to python-requests.org

### ISSUE THREAD

**Issue by domdfcoding** [psf/requests#6142]
Follow up to #6140 (which is locked)

In the sidebar the links to the docs in different languages point to e.g. `fr.python-requests.org`. These are specified in these files:

* https://github.com/psf/requests/blob/79f2ec3acc4e24fef6e6ce31ad7b1d4e2f77be31/docs/_templates/sidebarintro.html#L65
* https://github.com/psf/requests/blob/79f2ec3acc4e24fef6e6ce31ad7b1d4e2f77be31/docs/_templates/sidebarlogo.html#L60

While there are translations of the docs on readthedocs, at least [the German docs](https://requests.readthedocs.io/de/latest/) are woefully out of date. It might be better just to remove that section of the sidebar entirely.

----------------

The search box in the sidebar (which uses algolia search) has its links pointing to pages under `https://2.python-requests.org/en/master`. I can't find where this is configured. As it stands the search is non-functional.

**Comment by codingfiddler** [comment]
<img width="1169" alt="Screen Shot 2022-05-26 at 8 41 08 AM" src="https://user-images.githubusercontent.com/16449500/170499409-9b2b039b-528e-4205-93b6-877fcd8f402c.png">
Just a quick note that the old domain (now with a NET::ERR_CERT_COMMON_NAME_INVALID issue too) is still super high in search results and is likely going to cause some confusion. We can certainly get where we need to go, but I do hope this is helpful!

**Comment by nateprewitt** [comment]
Thanks for bringing this to our attention @domdfcoding. I've added #6144 which will be the first step in improving some of this. Many of those translations are now 8-10 years old, and given no one has volunteered to make updates in that window, I believe they're likely to remain that way.

I've removed the translations links from the sidebar as well as the algolia search. There should still be the quicksearch bar at the bottom of the sidebar for now and we'll want to look at leveraging RTDs search functionality going forward.
